### ISSUE TITLE
There are 2 places that need to be modified

### ISSUE THREAD

**Issue by lioumangdean** [heilig-defense/anti-fp#1]
![image](https://user-images.githubusercontent.com/44220609/121579631-785c8d00-ca5e-11eb-9cec-0d13362dd4df.png)
They are different in different browsers
![image](https://user-images.githubusercontent.com/44220609/121579882-bb1e6500-ca5e-11eb-94ff-c7f5c6627bbf.png)
![image](https://user-images.githubusercontent.com/44220609/121580247-19e3de80-ca5f-11eb-9cbb-399e452d047f.png)
Requests in chrome will issue such a unique request header. Firefox is similar.
Regarding font simulation, many tracking websites not only print the font list through API, they will try to load them in the background of the web page. If the fonts in the list cannot be displayed normally, they will be recognized quickly, so this aspect needs to be further considered.

**Comment by heilig-defense** [comment]
Anti-Fp only has basic capabilities against sec-* headers at this point.  It's already on our to-do list.

Not sure what you are trying to say about WebGL.  We know the values are different in different browsers.  Can you explain more what you expect?

Finally, you misunderstand how font tracking works and how Anti-Fp blocks it.  There is no "normal" and the act of measuring can be spoofed.  If you think otherwise, please provide a real tracking site that uses fonts as an input so we can test.

**Comment by lioumangdean** [comment]
Regarding WebGL, what I mean is that I want to be able to add fingerprints to the settings, because they are unique in disguising as different browsers. For example, I use chrome to pretend to be Firefox, and these values ​​need to be modified to look closer to a fake browser.
Regarding fonts, https://browserleaks.com/fonts
It can be seen that the test not only checks the list of fonts, but also renders the font randomly to determine whether the font actually exists. Secondly, I found that even with the same hardware and from different users, there will be many differences between fonts,

**Comment by heilig-defense** [comment]
Ok.  That's more clear.  We will look at expanding the available WebGL spoofing fields.

For fonts, the browserleaks font code doesn't check things randomly.  It simply has a large list of known font names and families and tests the returned dimensions for each one.  If the returned dimensions are unique, then that generally indicates that the font is installed. 

Fonts are not hardware dependent.  It's based on fonts on the system which can vary based on the software that has been installed.  I'm still a little confused about what the actual issue is.  Are you saying that the same Anti-Fp setting (for example Windows 7) produces different results on different machines?

**Comment by lioumangdean** [comment]
No, about fonts, I mean need more subtle choices, not to set fonts of different systems in general, because I found that even if the system is the same, there are many differences, which can produce more camouflage environments.
There is an interesting website here. https://pixelscan.net/.No matter how close my forgery is to the real user, it can always judge the disguise. Although it did not list a detailed checklist, and completed the judgment internally, which is very magical. Maybe you can refer to its testing items for certain optimization.

**Comment by heilig-defense** [comment]
WebGL updates were added to the last release along with some basic sec-* header spoofing.

We will leave this open while we work on new font features.
